replicated-security/noble-1/README.md

```diff
@@ -0,0 +1,91 @@
+
+![Strangelove logo](https://raw.githubusercontent.com/strangelove-ventures/goc-public/main/sl.png)
+# Noble Chain Information
+
+
+The `noble-1` chain will be launched as a persistent dummy chain to test basic Interchain Security functionality.
+
+* **Chain-ID**: `noble-1`
+* **denom**: `unoble`
```

Comment:
The genesis file only lists `usdc` denoms in the bank module, just making sure this is what you want.

Reply:
Yes